doc/source/api/alleleAnnotations.rst

```diff
@@ -0,0 +1,93 @@
+
+Variant Annotation API
+!!!!!!!!!!!!!!!!!!!!!!
+
+See `Allele Annotation schema <../schemas/alleleAnnotations.html>`_ for a detailed reference.
```

Comment:
I'm probably being dense, but what is the distinction between `allele` and `variant` here? They seem to be used more-or-less interchangeably in this documentation. Why `Variant Annotation API` and then the `Allele Annotation schema`? If they are interchangeable, perhaps it would be better to standardise on one (Variant?)?  If not, I think we need some discussion at the start to clarify this point.